pkgs/top-level/all-packages.nix

```diff
@@ -38335,6 +38335,8 @@ with pkgs;
 
   sfrotz = callPackage ../games/sfrotz { };
 
+  sgfutils = callPackage ../games/sgfutils { };
```

Comment:
Move this package into `pkgs/by-name/sg/sgfutils/package.nix` instead and drop the change to this file.

https://github.com/NixOS/nixpkgs/blob/master/pkgs/by-name/README.md